### ISSUE TITLE
Not compatible with Firefox Preview 75 (Nightly) on Android

### ISSUE THREAD

**Issue by sideshowbarker** [iamadamdev/bypass-paywalls-firefox#484]
In Firefox Preview 75 (Nightly) on Android, the browser disables Bypass Paywalls and shows:

> Firefox add-on technology is modernizing. These add-ons use frameworks that are not compatible with Firefox 75 & beyond.

Firefox version info shown in About Firefox Nightly:

Nightly 200214 08:03 (Build #2015722947)
32.0.0, 9ff30b22e
GV: 75.0a1-20200213035745

**Comment by iamadamdev** [comment]
Firefox Preview is not supported, but thanks for the heads up.

**Comment by sideshowbarker** [comment]
> Firefox Preview is not supported, but thanks for the heads up.

Yeah — I figured as much, but also figured I should take the time to raise an issue for it, in case it wasn’t already on record anywhere. (And by the way, massive thanks for Bypass Firewalls. I wish I could give the repo a thousand more stars.)
